modules/subnet/subnet.tf

```diff
@@ -23,7 +25,8 @@ resource "oci_core_subnet" "vcn_subnet" {
   #prohibit_internet_ingress  = var.enable_ipv6 && lookup(each.value,"type","public") == "public" ? each.value.prohibit_internet_ingress : false
   prohibit_public_ip_on_vnic = lookup(each.value, "type", "public") == "public" ? false : true
   route_table_id             = lookup(each.value, "type", "public") == "public" ? var.ig_route_id : var.nat_route_id
-  security_list_ids          = null
+  security_list_ids = lookup(each.value, "security_list_ids", null)
```

Comment:
The module creates a VCN and optionally some subnets. In these subnets, you want to attach security lists which can only be created after the VCN is created. 

But I don't see any security lists being created. Am I missing something?

Reply:
Yes , the interpretation of this request from my side was , the security lists will be added if a VCN exists . If a VCN does not exist, the user will not be able to add the security list . In terraform.tfvars.example line 106 will be relevant to the user's specific implementation if a VCN exists . Please let me know what you think

Reply:
We are not checking if a VCN exists here. This module is for creating a VCN. The ocids will only be available after the VCN is created

Reply:
@hyder - As this module is for creating VCN and the security lists do not exist before VCN creation is done, so it cannot handle attaching security lists to subnet that just got created as part of VCN creation. So, i think, this request can be rejected.